core/spec/models/spree/calculator/refunds/default_refund_amount_spec.rb

```diff
@@ -36,7 +36,7 @@
         order.adjustments.first.update_attributes(amount: adjustment_amount)
       end
 
-      it { is_expected.to eq line_item_price - (adjustment_amount.abs / line_item_quantity) }
+      it { is_expected.to be_within(0.1).of(line_item_price - (adjustment_amount.abs / line_item_quantity)) }
```

Comment:
The problem is that the existing code use a percentage to calculate the value of each line item. This works without problem when quantity is factor of 100 , but in other cases this returns a repeating decimal percentage and the result is a little bit different.

Ex:
existing code: 

```
def percentage_of_line_item(inventory_unit)
  1 / BigDecimal.new(inventory_unit) 
end
...

30 * percentage_of_line_item(3) # 30 * 0.333333333333333333 = 9.99999999999999999
```
what we want is 10 not 9.99999999999999999

with fix we have:

```
30 / 3 # 10 
```

Replaced the test from `/ line_item_quantity` to `* ( line_item_price / line_item_total )` because the code https://github.com/nebulab/solidus/pull/4/files#diff-f271b0c846fcbc382f43fdfa7f6a0c89R24 use a percentage instead a division